repo_resource/common.py

```diff
@@ -267,10 +267,10 @@ def __add_remote_revision(self, remote, rev):
         self.__remote_revision[remote] = rev
 
     def __get_remote_url(self, remote):
-        return self.__remote_url[remote]
+        return self.__remote_url.get(remote, None)
```

Comment:
Default value is None if value isn't found in Dict, so no need for the `None` parameter:
https://docs.python.org/3/library/stdtypes.html#dict.get

I don't know Google repo's behaviour if no remote is defined for the project in the manifest. In any case, we use this in the check step only, to compute the manifest version, so it's fine with me.